### ISSUE TITLE
[BUG] Chia Parallel Plots Deadlock Entire System(s)

### ISSUE THREAD

**Issue by SarahSzabo** [Chia-Network/chia-blockchain#6184]
Chia freezes all cores and deadlocks the entire system on Manjaro Linux. This does not happen on Ubuntu, but does on several of my Manjaro machines each with vastly different hardware.

Note that chia was NOT installed from AUR. It was installed using the standard process outlined on this github. This should be fairly easy to reproduce with a reasonable number of plots in parallel. It may have to plot for several hours for the condition to occurr however. If you decide to reproduce using fast NVMEs this may take more that 1 or even 2 plotting cycles.

**To Reproduce**
Use Chia command line -> start plots in parallel (this is not a RAM / Swap issue) -> wait a few hours -> come back -> observe system inoperable.

**Expected behavior**
Plots complete lol

**Screenshots**
I have better, journal logs from the last boot (notice the deadlock spin condition at the end):
[JournalLog.txt (Most Recent)](https://github.com/Chia-Network/chia-blockchain/files/6562444/JournalLog.txt)
[CHIA_Error_Full.txt (Previous)](https://github.com/Chia-Network/chia-blockchain/files/6562479/CHIA_Error_Full.txt)

![image](https://user-images.githubusercontent.com/37841645/120020962-71427300-bfb8-11eb-9021-39c025126c13.png)

**Desktop (please complete the following information):**

**Poweredge Server:**

```
[sarah@DawnSkyFoundry ~]$ inxi -Fx
System:    Host: DawnSkyFoundry Kernel: 5.10.36-2-MANJARO x86_64 bits: 64 compiler: gcc v: 10.2.0 Desktop: KDE Plasma 5.21.5 
           Distro: Manjaro Linux base: Arch Linux 
Machine:   Type: Server System: Dell product: PowerEdge R7425 v: N/A serial: <superuser required> 
           Mobo: Dell model: 02MJ3T v: X30 serial: <superuser required> UEFI: Dell v: 1.15.0 date: 09/11/2020 
CPU:       Info: 2x 32-Core (4-Die) model: AMD EPYC 7601 bits: 64 type: MT MCP MCM SMP arch: Zen rev: 2 cache: L2: 32 MiB 
           flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm bogomips: 562213 
           Speed: 1196 MHz min/max: 1200/2200 MHz boost: enabled Core speeds (MHz): 1: 1502 2: 1195 3: 1196 4: 1195 5: 1696 
           6: 1197 7: 1417 8: 1197 9: 1196 10: 1196 11: 1196 12: 1197 13: 1195 14: 1197 15: 1693 16: 1195 17: 1292 18: 1196 
           19: 1195 20: 1129 21: 1195 22: 1129 23: 1197 24: 2565 25: 1196 26: 1196 27: 1196 28: 1196 29: 1196 30: 1196 
           31: 1196 32: 1130 33: 1195 34: 1195 35: 1130 36: 1196 37: 1298 38: 1195 39: 1197 40: 1695 41: 1195 42: 1196 
           43: 1130 44: 1196 45: 1595 46: 1195 47: 1195 48: 1197 49: 1195 50: 1196 51: 1196 52: 1129 53: 1128 54: 1196 
           55: 1294 56: 1196 57: 1195 58: 1129 59: 1195 60: 1695 61: 1196 62: 1417 63: 1195 64: 1195 65: 1195 66: 1195 
           67: 1196 68: 1196 69: 1195 70: 1695 71: 1195 72: 1299 73: 1196 74: 1129 75: 1196 76: 1129 77: 1196 78: 1129 
           79: 1195 80: 2564 81: 1194 82: 1197 83: 1196 84: 1195 85: 1196 86: 1195 87: 1196 88: 1129 89: 1197 90: 1821 
           91: 1196 92: 1196 93: 1129 94: 1197 95: 1130 96: 1419 97: 1197 98: 1419 99: 1196 100: 1195 101: 1196 102: 1197 
           103: 1196 104: 1195 105: 1129 106: 1198 107: 1198 108: 1302 109: 1194 110: 1196 111: 1693 112: 1196 113: 1130 
           114: 1196 115: 1593 116: 1195 117: 1197 118: 1195 119: 1195 120: 1196 121: 1196 122: 1195 123: 1130 124: 1195 
           125: 1297 126: 1196 127: 1195 128: 1129 
Graphics:  Device-1: Matrox Systems Integrated Matrox G200eW3 Graphics driver: mgag200 v: kernel bus-ID: 03:00.0 
           Display: x11 server: X.Org 1.20.11 driver: loaded: modesetting resolution: 1600x900~60Hz 
           OpenGL: renderer: llvmpipe (LLVM 11.1.0 256 bits) v: 4.5 Mesa 21.0.3 direct render: Yes 
Audio:     Message: No device data found. 
           Sound Server-1: JACK v: 0.125.0 running: no 
           Sound Server-2: PulseAudio v: 14.2 running: yes 
           Sound Server-3: PipeWire v: 0.3.28 running: no 
Network:   Device-1: Intel I350 Gigabit Network vendor: Dell 4P I350-t rNDC driver: igb v: kernel port: N/A bus-ID: 01:00.0 
           IF: eno1 state: down mac: b8:ca:3a:64:a4:b8 
           Device-2: Intel I350 Gigabit Network vendor: Dell 4P I350-t rNDC driver: igb v: kernel port: N/A bus-ID: 01:00.1 
           IF: eno2 state: down mac: b8:ca:3a:64:a4:b9 
           Device-3: Intel I350 Gigabit Network vendor: Dell 4P I350-t rNDC driver: igb v: kernel port: N/A bus-ID: 01:00.2 
           IF: eno3 state: down mac: b8:ca:3a:64:a4:ba 
           Device-4: Intel I350 Gigabit Network vendor: Dell 4P I350-t rNDC driver: igb v: kernel port: N/A bus-ID: 01:00.3 
           IF: eno4 state: down mac: b8:ca:3a:64:a4:bb 
           Device-5: Intel 82599ES 10-Gigabit SFI/SFP+ Network driver: ixgbe v: kernel port: 4020 bus-ID: 41:00.0 
           IF: enp65s0f0 state: down mac: 80:61:5f:0a:35:66 
           Device-6: Intel 82599ES 10-Gigabit SFI/SFP+ Network driver: ixgbe v: kernel port: 4000 bus-ID: 41:00.1 
           IF: enp65s0f1 state: up speed: 10000 Mbps duplex: full mac: 80:61:5f:0a:35:67 
Drives:    Local Storage: total: 88.51 TiB used: 8.64 GiB (0.0%) 
           ID-1: /dev/nvme0n1 vendor: Western Digital model: WDS100T2B0C-00PXH0 size: 931.51 GiB temp: 26.9 C 
           ID-2: /dev/sda vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-3: /dev/sdaa vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-4: /dev/sdab vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-5: /dev/sdac vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-6: /dev/sdad vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-7: /dev/sdae vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-8: /dev/sdaf vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-9: /dev/sdag vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-10: /dev/sdah vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-11: /dev/sdai vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-12: /dev/sdaj vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-13: /dev/sdak vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-14: /dev/sdal vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-15: /dev/sdam vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-16: /dev/sdan vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-17: /dev/sdao vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-18: /dev/sdap vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-19: /dev/sdaq vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-20: /dev/sdar vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-21: /dev/sdas vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-22: /dev/sdat vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-23: /dev/sdau vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-24: /dev/sdav vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-25: /dev/sdaw vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-26: /dev/sdax vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-27: /dev/sday vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-28: /dev/sdaz vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-29: /dev/sdb vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-30: /dev/sdba vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-31: /dev/sdbb vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-32: /dev/sdbc vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-33: /dev/sdbd vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-34: /dev/sdbe vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-35: /dev/sdbf vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-36: /dev/sdbg vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-37: /dev/sdbh vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-38: /dev/sdbi vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-39: /dev/sdbj vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-40: /dev/sdbk vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-41: /dev/sdbl vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-42: /dev/sdbm vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-43: /dev/sdbn vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-44: /dev/sdbo vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-45: /dev/sdbp vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-46: /dev/sdbq vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-47: /dev/sdbr vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-48: /dev/sdbs vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-49: /dev/sdbt vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-50: /dev/sdbu vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-51: /dev/sdbv vendor: Hitachi model: HUC106030CSS601 size: 279.4 GiB 
           ID-52: /dev/sdbw vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-53: /dev/sdbx vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-54: /dev/sdby vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-55: /dev/sdbz vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-56: /dev/sdc vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-57: /dev/sdca vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-58: /dev/sdcb vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-59: /dev/sdcc vendor: Hitachi model: HUC106030CSS601 size: 279.4 GiB 
           ID-60: /dev/sdcd vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-61: /dev/sdce vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-62: /dev/sdcf vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-63: /dev/sdcg vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-64: /dev/sdch vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-65: /dev/sdci vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-66: /dev/sdcj vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-67: /dev/sdck vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-68: /dev/sdcl vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-69: /dev/sdcm vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-70: /dev/sdcn vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-71: /dev/sdco vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-72: /dev/sdcp vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-73: /dev/sdcq vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-74: /dev/sdcr vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-75: /dev/sdcs vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-76: /dev/sdct vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-77: /dev/sdcu vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-78: /dev/sdcv vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-79: /dev/sdcw vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-80: /dev/sdcx vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-81: /dev/sdcy vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-82: /dev/sdcz vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-83: /dev/sdd vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-84: /dev/sdda vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-85: /dev/sddb vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-86: /dev/sddc vendor: Hitachi model: HUC106030CSS601 size: 279.4 GiB 
           ID-87: /dev/sddd vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-88: /dev/sdde vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-89: /dev/sddf vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-90: /dev/sddg vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-91: /dev/sde vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-92: /dev/sdf vendor: Hitachi model: HUS726060AL5210 size: 5.46 TiB 
           ID-93: /dev/sdg vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-94: /dev/sdh vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-95: /dev/sdi vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-96: /dev/sdj vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-97: /dev/sdk vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-98: /dev/sdl vendor: Hitachi model: HUS724040ALS641 size: 3.64 TiB 
           ID-99: /dev/sdm vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-100: /dev/sdn vendor: Hitachi model: HUS724040ALS641 size: 3.64 TiB 
           ID-101: /dev/sdo vendor: Hitachi model: HUS724040ALS640 size: 3.64 TiB 
           ID-102: /dev/sdp vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-103: /dev/sdq vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-104: /dev/sdr vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-105: /dev/sds vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-106: /dev/sdt vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-107: /dev/sdu vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-108: /dev/sdv vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-109: /dev/sdw vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-110: /dev/sdx vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-111: /dev/sdy vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
           ID-112: /dev/sdz vendor: Hitachi model: HUC109030CSS601 size: 279.4 GiB 
Partition: ID-1: / size: 512 GiB used: 8.64 GiB (1.7%) fs: btrfs dev: /dev/nvme0n1p2 
           ID-2: /boot/efi size: 511 MiB used: 576 KiB (0.1%) fs: vfat dev: /dev/nvme0n1p1 
           ID-3: /home size: 512 GiB used: 8.64 GiB (1.7%) fs: btrfs dev: /dev/nvme0n1p2 
           ID-4: /var/log size: 512 GiB used: 8.64 GiB (1.7%) fs: btrfs dev: /dev/nvme0n1p2 
Swap:      ID-1: swap-1 type: partition size: 419.01 GiB used: 0 KiB (0.0%) dev: /dev/nvme0n1p3 
Sensors:   System Temperatures: cpu: 27.5 C mobo: 0 C 
           Fan Speeds (RPM): N/A 
Info:      Processes: 1405 Uptime: 14m Memory: 503.63 GiB used: 5.54 GiB (1.1%) Init: systemd Compilers: gcc: 10.2.0 
           Packages: 1223 Shell: Bash v: 5.1.8 inxi: 3.3.04
```

**Additional context**

**This also happens on my upstairs computer:**

```
System:
  Host: MidnightStarSign Kernel: 5.10.36-2-MANJARO x86_64 bits: 64 compiler: gcc 
  v: 10.2.0 Desktop: KDE Plasma 5.21.5 Distro: Manjaro Linux base: Arch Linux 
Machine:
  Type: Desktop Mobo: ASUSTeK model: PRIME X570-PRO v: Rev X.0x 
  serial: <superuser required> UEFI: American Megatrends v: 3001 date: 12/04/2020 
CPU:
  Info: 16-Core model: AMD Ryzen 9 5950X bits: 64 type: MT MCP arch: Zen 3 rev: 0 
  cache: L2: 8 MiB 
  flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm 
  bogomips: 217690 
  Speed: 3333 MHz min/max: 2200/3400 MHz boost: enabled Core speeds (MHz): 1: 3333 
  2: 2585 3: 2200 4: 2199 5: 2200 6: 2199 7: 2199 8: 2199 9: 2841 10: 3067 11: 3154 
  12: 2808 13: 3579 14: 3410 15: 2383 16: 3172 17: 2797 18: 2200 19: 2200 20: 2199 
  21: 2200 22: 2199 23: 2199 24: 2715 25: 2200 26: 2200 27: 2634 28: 3254 29: 2723 
  30: 3018 31: 2199 32: 2200 
Graphics:
  Device-1: NVIDIA GA104 [GeForce RTX 3070] vendor: ASUSTeK driver: nvidia v: 460.80 
  bus-ID: 0a:00.0 
  Device-2: Microdia USB 2.0 Camera type: USB driver: snd-usb-audio,uvcvideo 
  bus-ID: 7-1:2 
  Display: x11 server: X.Org 1.20.11 driver: loaded: nvidia resolution: 
  1: 1920x1080~60Hz 2: 1920x1080 3: 1920x1080 
  OpenGL: renderer: GeForce RTX 3070/PCIe/SSE2 v: 4.6.0 NVIDIA 460.80 
  direct render: Yes 
Audio:
  Device-1: NVIDIA vendor: ASUSTeK driver: snd_hda_intel v: kernel bus-ID: 0a:00.1 
  Device-2: AMD Starship/Matisse HD Audio vendor: ASUSTeK driver: snd_hda_intel 
  v: kernel bus-ID: 0c:00.4 
  Device-3: JMTek LLC. Plugable USB Audio Device type: USB 
  driver: hid-generic,snd-usb-audio,usbhid bus-ID: 3-1:2 
  Device-4: Schiit Audio Schiit Modi 3+ type: USB driver: snd-usb-audio 
  bus-ID: 3-2:3 
  Device-5: ASUSTek ASUS AI Noise-Cancelling Mic Adapter type: USB 
  driver: hid-generic,snd-usb-audio,usbhid bus-ID: 5-5:3 
  Device-6: Microdia USB 2.0 Camera type: USB driver: snd-usb-audio,uvcvideo 
  bus-ID: 7-1:2 
  Sound Server-1: ALSA v: k5.10.36-2-MANJARO running: yes 
  Sound Server-2: JACK v: 0.125.0 running: no 
  Sound Server-3: PulseAudio v: 14.2 running: yes 
  Sound Server-4: PipeWire v: 0.3.28 running: yes 
Network:
  Device-1: Intel I211 Gigabit Network vendor: ASUSTeK driver: igb v: kernel 
  port: f000 bus-ID: 06:00.0 
  IF: enp6s0 state: up speed: 1000 Mbps duplex: full mac: 24:4b:fe:5b:08:2a 
Bluetooth:
  Device-1: Cambridge Silicon Radio Bluetooth Dongle (HCI mode) type: USB 
  driver: btusb v: 0.8 bus-ID: 3-5.3:6 
  Report: rfkill ID: hci0 rfk-id: 0 state: up address: see --recommends 
Drives:
  Local Storage: total: 3.69 TiB used: 1.74 TiB (47.2%) 
  ID-1: /dev/nvme0n1 vendor: Western Digital model: WDS100T3X0C-00SJG0 
  size: 931.51 GiB temp: 48.9 C 
  ID-2: /dev/nvme1n1 vendor: Western Digital model: WDS100T2B0C-00PXH0 
  size: 931.51 GiB temp: 41.9 C 
  ID-3: /dev/sda vendor: Seagate model: ST2000LM015-2E8174 size: 1.82 TiB 
  ID-4: /dev/sdb type: USB vendor: SanDisk model: Gaming Xbox 360 size: 7.48 GiB 
  ID-5: /dev/sdc type: USB vendor: Toshiba model: TransMemory size: 14.92 GiB 
  ID-6: /dev/sdd type: USB model: N/A size: 29.3 GiB 
Partition:
  ID-1: / size: 767 GiB used: 653.55 GiB (85.2%) fs: btrfs dev: /dev/dm-0 
  mapped: luks-466d5812-64c7-4a28-bcc4-a1a5adfa9450 
  ID-2: /boot/efi size: 511 MiB used: 26.1 MiB (5.1%) fs: vfat dev: /dev/nvme0n1p1 
  ID-3: /home size: 767 GiB used: 653.55 GiB (85.2%) fs: btrfs dev: /dev/dm-0 
  mapped: luks-466d5812-64c7-4a28-bcc4-a1a5adfa9450 
Swap:
  ID-1: swap-1 type: partition size: 64 GiB used: 9.9 GiB (15.5%) dev: /dev/dm-1 
  mapped: luks-81b2dc57-06f5-4471-b484-77c3a516f307 
Sensors:
  System Temperatures: cpu: 57.5 C mobo: 0 C gpu: nvidia temp: 41 C 
  Fan Speeds (RPM): N/A gpu: nvidia fan: 0% 
Info:
  Processes: 1058 Uptime: 7d 3h 31m Memory: 62.78 GiB used: 43.95 GiB (70.0%) 
  Init: systemd Compilers: gcc: 10.2.0 clang: 11.1.0 Packages: 1991 Shell: Bash 
  v: 5.1.8 inxi: 3.3.04 

```

**Comment by baizuchong-net** [comment]
这配置不错

**Comment by SarahSzabo** [comment]
The issue in my testing appears related to the most recent LTS kernel

**Comment by mychiux413** [comment]
I have same equipment like yours,
I used  `R9 5950` + `kernel 5.10` + `ubuntu` to plot,
which would definitely crash at some point,
after crash, my SSD would be also broken, and I have to reformat it then.

My temporary solution is to downgrade the linux kernel to 5.8.0,
which could make my plotting more stable,
but..., which could also raise another issue that `dmesg` will show you:
"EDAC amd64: Error: F0 not found, device 0x1650 (broken BIOS?)"
this issue should be fixed after kernel 5.10 (see [buzilla](https://bugzilla.kernel.org/show_bug.cgi?id=210593))
and I don't know will such issue drag down my plotting speed.
